recipe/meta.yaml

```diff
@@ -1,5 +1,5 @@
 {% set name = "conda-forge-ci-setup" %}
-{% set version = "3.24.0" %}
+{% set version = "3.24.1" %}
```

Comment:
Forgot PR ( https://github.com/conda-forge/conda-forge-ci-setup-feedstock/pull/217 ) used `3.24.1` recently, bumped to `3.24.2` in commit ( https://github.com/conda-forge/conda-forge-ci-setup-feedstock/commit/2b1d2c7835132dd36ca061c538f2904f9a4646d7 )